doc/source/serve/multi-app.md

```diff
@@ -106,50 +116,53 @@ Check the status of the applications by running `serve status`.
 ```console
 $ serve status
 proxies:
-  59e6321aa1699fa09ed1094e1f0b3c37afd47fc208802db3c07328d0: HEALTHY
+  2e02a03ad64b3f3810b0dd6c3265c8a00ac36c13b2b0937cbf1ef153: HEALTHY
 applications:
-  app1:
+  classify:
     status: RUNNING
     message: ''
-    last_deployed_time_s: 1693428138.3633459
+    last_deployed_time_s: 1693267064.0735464
     deployments:
-      Adder:
+      downloader:
         status: HEALTHY
         replica_states:
           RUNNING: 1
         message: ''
-      Multiplier:
+      ImageClassifier:
         status: HEALTHY
         replica_states:
           RUNNING: 1
         message: ''
-      Router:
-        status: HEALTHY
-        replica_states:
-          RUNNING: 1
-        message: ''
-      DAGDriver:
-        status: HEALTHY
-        replica_states:
-          RUNNING: 1
-        message: ''
-  app2:
+  translate:
     status: RUNNING
     message: ''
-    last_deployed_time_s: 1693428138.3633459
+    last_deployed_time_s: 1693267064.0735464
     deployments:
-      greet:
-        status: HEALTHY
-        replica_states:
-          RUNNING: 1
-        message: ''
-      DAGDriver:
+      Translator:
         status: HEALTHY
         replica_states:
           RUNNING: 1
         message: ''
 ```
 
+### Send requests between applications
+You can also make calls between applications without going through HTTP by using the Serve API `serve.get_app_handle` to get a handle to any live Serve application on the cluster. This handle can be used to directly execute a request on an application. Take the classifier and translator app above as an example. We can modify the `__call__` method of the `ImageClassifier` to check for another parameter in the HTTP request, and send requests to the translator application.
+
+```{literalinclude} doc_code/image_classifier_example.py
+:language: python
+:start-after: __serve_example_modified_begin__
+:end-before: __serve_example_modified_end__
+```
+
+Then, sending requests to the classifier application with the `should_translate` flag set to True:
+```python
+import requests
+
+requests.post("http://localhost:8000/classify", json={"image_url": "https://cdn.britannica.com/41/156441-050-A4424AEC/Grizzly-bear-Jasper-National-Park-Canada-Alberta.jpg", "should_translate": False}).text
+'Braunbär, Bruin, Ursus arctos'
```

Comment:
I'm kinda confused, should this be in English since `should_translate` is false?

Reply:
Or maybe `should_translate` is meant to be true?

Reply:
Yeah it should be true, fixed